solidity/contracts/token/TokenBridgeCctp.sol

```diff
@@ -85,6 +85,14 @@ contract TokenBridgeCctp is HypERC20Collateral, AbstractCcipReadIsm {
         wrappedToken.approve(address(tokenMessenger), type(uint256).max);
     }
 
+    function initialize(
+        address _hook,
+        address _interchainSecurityModule,
+        address _owner
+    ) public override {
+        revert("Only one initialize() function is allowed");
```

Comment:
nit: make it clear which initialize should be called

Reply:
Done